lambda/getAllSimUsageWirelessLogic.ts

```diff
@@ -50,13 +60,28 @@ export const handler = async (): Promise<void> => {
 		console.error(usage.error)
 		return
 	}
+	const records: _Record[] = []
 	await Promise.all(
 		iccids.map(async (iccid) => {
+			const diff =
+				(iccidAndUsage[iccid] ?? 0) - (usage.value.usedBytes[iccid] ?? 0)
+			records.push(usageToRecord({ iccid, diff }).record)
 			const simDetails = {
 				usedBytes: usage.value.usedBytes[iccid] ?? 0,
 				totalBytes: usage.value.totalBytes,
 			}
-			await putSimDetailsFunc(iccid, true, simDetails)
+			await putSimDetailsFunc({ iccid, simExisting: true, simDetails })
 		}),
 	)
+	const historicalDataStoring = await storeHistoricalData(records)
+	if ('error' in historicalDataStoring) {
+		if (historicalDataStoring.error instanceof RejectedRecordsException) {
+			console.debug(
```

Comment:
```suggestion
			console.error(
```

Should show up as an error in the logs so you can create an alert for these.